src/test/scala/elevate/fsmooth/rewriteExamples.scala

```diff
@@ -64,7 +64,7 @@ class rewriteExamples extends elevate.test_util.Tests {
     println(step3)
 
     println("\n-- step4: partial evaluation + simplification")
-    val strategy3 = normalize.apply(letPartialEvaluation <+ additionZero)
+    val strategy3 = normalize(traversal.FSmoothTraversable)(letPartialEvaluation <+ additionZero)
```

Comment:
the implicit does not work well anymore with currying I assume?

Reply:
Yeah, they work differently (for now).